src/server.js

```diff
@@ -79,7 +78,7 @@ module.exports = function (classes) {
       _checkAuth: function (req) {
         var self = this;
 
-        if (self.authHandler) {
+        if (self.authType !== Authorization.NONE) {
```

Comment:
shouldn't this become an argument in _checkAuth? (authType instead of a prototype member)